common/lib/xmodule/xmodule/modulestore/tests/test_split_modulestore.py

```diff
@@ -888,6 +889,25 @@ def test_block_generations(self, _from_json):
         version_history = modulestore().get_block_generations(second_problem.location)
         self.assertNotEqual(version_history.locator.version_guid, first_problem.location.version_guid)
 
+    def test_make_course_usage_key(self):
```

Comment:
hmm.. didn't realize CCX doesn't work with old-mongo.  so your ddt in test_mixed_modulestore would need to be explicit about which CourseKey types are tested with which modulestore types.